### ISSUE TITLE
IMPORTANT: Glitched SW Innkeeper/NPC's

### ISSUE THREAD

**Issue by ajberret** [dalaranwow/dalaran-wow#4619]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

Very often, a horde priest will come to SW, mind control the innkeeper and/or several other NPC's into the SW wall glitch, dropping them below the map where they won't respawn and rendering certain functions of SW useless, giving the game the appearance of being extremely bugged. This will deter new players who are testing your game, just like I did, and if the first thing they see after creating a lvl 1 human is the absence of several key SW NPC's that dont respawn, they will get the false impression that the server is trash and unkept/uncared for. 

**Expected behaviour**: Tell us what should happen instead

NPC's should respawn regardless of player interaction/be immune to wall glitch

**Steps to reproduce the problem**:

1. The video in step #2 shows in the first ten seconds how this glitch is obtained, all the horde priest has to do is mind control an npc and presto....
2. https://www.youtube.com/watch?v=55d-bas0hj4
3. Please make the NPCs in SW immune to Mind Control or fix this in some way. It's hurting the server's health and well being.

**Include proofs for this behaviour**
![image](https://cloud.githubusercontent.com/assets/13809154/24148624/0deb94ba-0e16-11e7-990f-7ab72c289432.png)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Teridan-dal-wow** [comment]
#4613

**Comment by ajberret** [comment]
Laavalambbu is one of the indiviudals constantly exploiting the glitch

**Comment by Rushor** [comment]
already temp-fixed in https://github.com/dalaranwow/dalaran-wow/issues/4613. The fix will be applied on the live realm soon.
